.github/workflows/canister-tests.yml

```diff
@@ -362,7 +404,7 @@ jobs:
           # a relatively small price to pay to make sure PRs are always tested against the latest release.
           curl -sSL https://github.com/dfinity/internet-identity/releases/latest/download/internet_identity_test.wasm.gz -o internet_identity_previous.wasm.gz
           curl -sSL https://github.com/dfinity/internet-identity/releases/latest/download/archive.wasm.gz -o archive_previous.wasm.gz
-          cargo test --release
+          ./cargo-nextest nextest run --archive-file canister-tests-${{ matrix.os }}.tar.zst --partition hash:${{ matrix.partition }}
```

Comment:
I didn't dig too deep in the docs, but shouldn't this be `--partition count:1/3`?

Reply:
They have `count` and `hash`. Using `hash` makes sure, that the partition is stable, even if tests are added or removed (but might lead to slightly worse balancing on few tests).
But I think having stable partitioning is beneficial and we have enough tests that the shards are balanced enough.